modin/pandas/test/dataframe/test_reduction.py

```diff
@@ -312,6 +312,38 @@ def test_prod_specific(min_count, numeric_only):
     )
 
 
+@pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
+@pytest.mark.parametrize("axis", axis_values, ids=axis_keys)
+@pytest.mark.parametrize(
+    "skipna", bool_arg_values, ids=arg_keys("skipna", bool_arg_keys)
+)
+@pytest.mark.parametrize(
+    "numeric_only", bool_arg_values, ids=arg_keys("numeric_only", bool_arg_keys)
+)
+@pytest.mark.parametrize("ddof", int_arg_values, ids=arg_keys("ddof", int_arg_keys))
```

Comment:
Parameter space too large. Performance will be hit hard.

Please divide into multiply tests.

For example, you can see `test_window.py::test_median_skew`

Reply:
Divided into multiple tests.